finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -99,10 +98,13 @@ private Properties getPropertiesWithGivenHostName(String hostname) {
     String customURIString = "http://" + hostname + ":" + DEFAULT_PORT + "/system";
     URI customURI = null;
     try {
-      customURI = new URI(customURIString);
+      //customURI = new URI(customURIString);
```

Comment:
I'd recommend removing this commented line.

Reply:
Good catch. Removed. Thank you!